<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameInfo</class>
 <widget class="QDialog" name="GameInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameInfo</string>
  </property>
  <property name="locale">
   <locale language="C" country="AnyCountry"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_GameInfo_Iterator">
     <item>
      <widget class="QPushButton" name="pushButton_GameInfo_First">
       <property name="text">
        <string>&lt;&lt;&amp;First</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_GameInfo_Previous">
       <property name="text">
        <string>&lt;&amp;Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_GameInfo_Iterator">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_GameInfo_Next">
       <property name="text">
        <string>&amp;Next&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_GameInfo_Last">
       <property name="text">
        <string>&amp;Last&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_GameInfo_Content">
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_GameInfo_SaveType">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Internal binary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Internal text</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Binary file</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Text file</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Multi file</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_GameInfo_Name">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_GameInfo_Name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_GameInfo_SaveType">
       <property name="text">
        <string>SaveType:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_GameInfo_OriginSavePath">
       <property name="text">
        <string>OriginSavePath:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_GameInfo_BackupUrls"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_GameInfo_PrimaryUrl">
       <property name="text">
        <string>PrimaryUrl:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_GameInfo_PrimaryUrl"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_GameInfo_BackupUrls">
       <property name="text">
        <string>BackupUrls:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_GameInfo_OriginSavePath"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_GameInfo_Memo">
       <property name="text">
        <string>Memo:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPlainTextEdit" name="plainTextEdit_GameInfo_Memo"/>
     </item>
     <item row="4" column="2">
      <widget class="QToolButton" name="toolButton_GameInfo_OriginSavePath">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_GameInfo_Lock">
       <property name="text">
        <string>Unlock</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_GameInfo_Bottom">
     <item>
      <spacer name="horizontalSpacer_GameInfo_BottomLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_GameInfo_Commit">
       <property name="text">
        <string>&amp;Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_GameInfo_Ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_GameInfo_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_GameInfo_BottomRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
